resources/assets/components/Signup/index.js

```diff
@@ -195,13 +194,21 @@ class Signup extends React.Component {
   }
 
   // Tag a post.
-  updateTag(postId, tag) {
+  updateTag(postId, tag, requestMethod) {
     const fields = {
       post_id: postId,
       tag_name: tag,
     };
 
-    const response = this.api.post(`api/v3/posts/${postId}/tags`, fields);
+    this.api = new RestApiClient(window.location.origin, {
+        headers: {
+          'Authorization' : `Bearer ${window.AUTH}`,
+          'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
(that same `422 (Unprocessable Entity)` error)

Reply:
That's so strange! I just tested out that endpoint on my local (via Paw) and it works with a JSON body.

Reply:
weird!!

Reply:
when looking in the Network tab, it doesn't look like it is sending over any form-data for some reason when doing the `DELETE` method 🤔

Reply:
@DFurnes Ah! Because gateway-js doesn't do this yet! I'll update [here](https://github.com/DoSomething/gateway-js/blob/master/src/RestApiClient.js#L52-L56) now :)

Reply:
PR [here](https://github.com/DoSomething/gateway-js/pull/17) :)